index.js

```diff
@@ -1,7 +1,9 @@
 module.exports = function xXssProtection (options) {
+  var checkReport = options && options.reportUri ? '1; report=' + options.reportUri : '1; mode=block'
```

Comment:
Nitpick: I think I'd prefer a variable name like `headerValue` for this.

Reply:
done